src/api/libcellml/component.h

```diff
@@ -37,12 +37,19 @@ class LIBCELLML_EXPORT Component: public ComponentEntity, public ImportedEntity
                                   public std::enable_shared_from_this<Component>
 #endif
 {
-public:
+private:
```

Comment:
FWIW, we are becoming increasingly inconsistent. Shouldn't we stick to having the `public`, `protected` and then `private` sections in that order?